packages/shared/src/components/search/SearchPanel/SearchPanel.tsx

```diff
@@ -43,10 +33,7 @@ export type SearchPanelClassName = {
 };
 
 export const SearchPanel = ({ className }: SearchPanelProps): ReactElement => {
-  useContext(SettingsContext);
```

Comment:
As far as I can tell, this doesn't seem to do anything? 🤔

Reply:
Nah doesn't look like it's needed for anything.